awx_collection/plugins/modules/tower_instance_group.py

```diff
@@ -92,6 +92,8 @@ def main():
         pod_spec_override=dict(),
         instances=dict(required=False, type="list", default=None),
         state=dict(choices=['present', 'absent'], default='present'),
+        is_containerized=dict(type='bool'),
```

Comment:
cc @AlanCoding @shanemcd (is this field name changing soon?)

Reply:
Checking the current API in `devel` branch

```json
            "capacity": 298,
            "committed_capacity": 0,
            "consumed_capacity": 0,
            "percent_capacity_remaining": 100.0,
            "jobs_running": 0,
            "jobs_total": 0,
            "instances": 1,
            "controller": null,
            "is_controller": false,
            "is_isolated": false,
            "is_container_group": false,
            "credential": null,
            "policy_instance_percentage": 100,
            "policy_instance_minimum": 0,
            "policy_instance_list": [],
            "pod_spec_override": "",
```

(for instance group)

So you are right @ryanpetrello, this field no longer exists